functional/unit_action_test.go

```diff
@@ -313,6 +313,9 @@ func replaceUnitCommon(cmd string) error {
 	if len(units) != 1 {
 		return fmt.Errorf("Did not find 1 unit in cluster: \n%s", stdout)
 	}
+	if err := waitForActiveUnitsReplaceCmds(cluster, m, cmd, 1); err != nil {
+		return err
+	}
```

Comment:
@antrik
> my point was that instead of a one-off check directly after the command, it is probably better to use the wait function to see whether the expected state is reached eventually... At least that's what all the existing tests I have looked at so far seem to be doing.

All right. I made that change, and tested it. It looks fine.
Pushed to my branch: https://github.com/endocode/fleet/tree/dongsu/fleetctl-replace-unit-fxtests
Hopefully that will be merged to this PR.